src/Modules/Reports/components/Hero.tsx

```diff
@@ -0,0 +1,72 @@
+import {
+  HStack,
+  Menu,
+  MenuButton,
+  MenuList,
+  MenuItem,
+  Box,
+  Text,
+} from "@chakra-ui/react";
+import React from "react";
+import { AiFillCaretDown } from "react-icons/ai";
+
+const Hero = () => {
+  return (
```

Comment:
this component does not look the same with what is on the ui. please check the ui